### DISCUSSION TITLE
Erro ao importar pacotes Sqlite e Design

### DISCUSSION THREAD

**Discussion by GersonSantiago** [balta-io/forum#6] [Tira Dúvidas] [answered]
Estou iniciando o curso Fundamentos do ASP.NET 6 e seguindo as orientações para importar os pacotes Sqlite e Design, ocorre este problema conforme imagem em anexo. Poderia me auxiliar ?

![image](https://user-images.githubusercontent.com/99631213/189944830-fd55f3f8-874d-4736-9019-b100188e7a84.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @GersonSantiago, como vai?

Poderia tentar limpar e adicionar os certificados do .NET novamente?
https://balta.io/blog/dotnet-instalacao-configuracao-e-primeiros-passos#atualizando-os-certificados

Eu tive este problema faz um mês e se não me engano resolvi com isto.
Em adicional, via CLI ele não estava funcionando na minha máquina (Provavelmente alguma versão do Nuget) mas via Visual Studio funcionava.

Então se o passo acima não funcionar, tenta atualizar o .NET e se mesmo assim não funcionar, tenta instalar por dentro do Visual Studio.

**Reply by GersonSantiago** [reply]
Obrigado professor, tudo certinho agora :)

**Reply by andrebaltieri** [reply]
Aeeee!!! <3
